services/storage-system/pkg/revaconfig/config.go

```diff
@@ -1,12 +1,25 @@
 package revaconfig
 
 import (
+	"strings"
+
 	userpb "github.com/cs3org/go-cs3apis/cs3/identity/user/v1beta1"
 	"github.com/owncloud/ocis/v2/services/storage-system/pkg/config"
 )
 
 // StorageSystemFromStruct will adapt an oCIS config struct into a reva mapstructure to start a reva service.
 func StorageSystemFromStruct(cfg *config.Config) map[string]interface{} {
+	self := cfg.GRPC.Addr
```

Comment:
Same here. Please don't write java :cry: Call it `grpcAddr` (or something else that describes what it is)